test/esp/resources/signature_test.rb

```diff
@@ -145,16 +145,16 @@ class SignatureTest < ActiveSupport::TestCase
             signature = ESP::Signature.first
             external_account_id = ESP::ExternalAccount.last.id
 
-            alerts = signature.run(external_account_id: external_account_id, regions: 'us_east_1')
+            alerts = signature.run(external_account_id: external_account_id, region: 'us_east_1')
 
             assert_equal ESP::Alert, alerts.resource_class
           end
 
           should 'return errors' do
             signature = ESP::Signature.first
-            external_account_id = 999
+            external_account_id = 999999999999
```

Comment:
Style/NumericLiterals: Separate every 3 digits in the integer portion of a number with underscores(_).